src/Rest/Routes/Settings/BulkRoute.php

```diff
@@ -224,7 +224,7 @@ private function output(array $details, string $type): array
 		if (\count($details) > 1) {
 			$msgOutput = [
 				// translators: %s replaces type.
-				\sprintf(\esc_html__('Not all forms were %s with success. Please check the following log.', 'eightshift-forms'), $intrernaType, $msg),
+				\sprintf(\esc_html__('Not all items were %s with success. Please check the following log.', 'eightshift-forms'), $intrernaType, $msg),
```

Comment:
```suggestion
				\sprintf(\esc_html__('Not all items were %s with success. Please check the following log.', 'eightshift-forms'), $intrernaType),
```

There is only one replacement in the `sprintf`. Or did you mean to add another for the `$msg` parameter?